kogito-editors-js/packages/feel-input-component/showcase/src/index.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import ReactDOM from "react-dom";
+import { FeelService } from "./FeelService";
+import "./index.css";
+import { FeelInput } from "./lib";
+
+const FeelEditor = () => {
+  const [feelExpression, setFeelExpression] = useState("");
+  const [feelResult, setFeelResult] = useState("");
+  const suggestionProvider = useCallback((text, row, col) => {
+    return FeelService.getInstance().getSuggestions(text, row, col);
+  }, []);
+
+  useEffect(() => {
+    const clientResult = FeelService.getInstance().evaluate(feelExpression);
+    if (clientResult !== "") {
+      setFeelResult(clientResult);
+    }
+
+    window.clearTimeout(window.__KIE__FEEL__THROTTLING___);
+    window.__KIE__FEEL__THROTTLING___ = window.setTimeout(() => {
+      const FEEL_SERVER = "https://piscine-madame-68280.herokuapp.com";
```

Comment:
I am not able to access this URL, why we use it please?

generally I think it is not an recommended approach to access third party URLs in your application.

Could you please explain a little bit more this url?

Reply:
Well spotted, @jomarko. I just removed this in favor of an optional property.

As I mentioned in the other comment, we will have the FEEL output component shortly. However, our FEEL client-side-parser is not as fully stable as the server-side one.

This URL points to a backend fallback (that will rely on DMN runner) to evaluate FEEL expressions that the FEEL client-side cannot process.